### ISSUE TITLE
Don't block media control keys

### ISSUE THREAD

**Issue by qwerty287** [slgobinath/SafeEyes#433]
Hello,
if SafeEyes appears on my screen and I'd like to pause a film or music, that's not possible because SafeEyes blocks the keyboard. Would it be possible to unblock only the media keys, so you can pause music etc.?

**Comment by slgobinath** [comment]
Hi, Yup its something to be fixed. Until then you can use the Media Control plugin. This plugin will show a pause button on the top right corner of the screen to pause any supported players.

**Comment by nifadyev** [comment]
Possibly related to #213

**Comment by deltragon** [comment]
This is a duplicate of #213.
